modules/programs/noti.nix

```diff
@@ -0,0 +1,55 @@
+{ config, lib, pkgs, ...}:
+
+with lib;
+
+let
+
+  cfg = config.programs.noti;
+
+in
+
+{
+  meta.maintainers = [ maintainers.marsam ];
+
+  options.programs.noti = {
+    enable = mkEnableOption "Noti";
+
+    settings = mkOption {
+      type = types.attrs;
+      default = {};
+      description = ''
+        Configuration written to
+        <filename>~/.config/noti/noti.yaml</filename>.
+
+        See
+        <citerefentry>
+          <refentrytitle>noti.yaml</refentrytitle>
+          <manvolnum>5</manvolnum>
+        </citerefentry>.
+        for the full list of options.
+      '';
+      example = {
```

Comment:
I think you'll want `literalExample` here to make the example show up a bit nicer in the documentation. I.e., something like
```nix
example = literalExample ''
  {
    say = {
      voice = "Alex";
    };
    slack = {
      token = "1234567890abcdefg";
      channel = "@jaime";
      username = "noti";
    };
  };
'';
```
Alternatively just shorten it to something like
```nix
example = { say = { voice = "Alex"; }; };
```